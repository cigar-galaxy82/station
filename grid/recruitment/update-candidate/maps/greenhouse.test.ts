import {
  updateCandidateTest,
} from './update-candidate';
import { updateCandidateFeaturesTest } from './update-candidate-features';

updateCandidateTest('greenhouse', {
  validCandidateId: '5884696006',
  invalidCandidateId: '-1',
});

updateCandidateFeaturesTest('greenhouse');