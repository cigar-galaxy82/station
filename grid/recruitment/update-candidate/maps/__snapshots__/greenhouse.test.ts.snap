// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`recruitment/update-candidate/greenhouse UpdateCandidate when specified candidate ID does not exist returns error 1`] = `
Err {
  "error": "HTTPError: HTTP Error
AST Path: definitions[0].statements[1]
Original Map Location: Line 11, column 3",
}
`;

exports[`recruitment/update-candidate/greenhouse UpdateCandidate when specified job does exist when valid inputs passed performs correctly 1`] = `
Ok {
  "value": undefined,
}
`;

exports[`recruitment/update-candidate/greenhouse UpdateCandidateFeatures returns valid greenhouse features 1`] = `
Ok {
  "value": Object {
    "cvMIMETypes": Array [
      "application/atom+xml",
      "application/javascript",
      "application/json",
      "application/msgpack",
      "application/msword",
      "application/pdf",
      "application/rss+xml",
      "application/vnd.ms-excel",
      "application/vnd.openxmlformats-officedocument.spreadsheetml.sheet",
      "application/vnd.openxmlformats-officedocument.wordprocessingml.document",
      "application/vnd.ms-powerpoint",
      "application/xml",
      "application/x-www-form-urlencoded",
      "application/x-yaml",
      "application/zip",
      "multipart/form-data",
      "image/bmp",
      "image/gif",
      "image/jpeg",
      "image/png",
      "image/tiff",
      "text/calendar",
      "text/css",
      "text/csv",
      "text/html",
      "text/javascript",
      "text/plain",
      "text/vcard",
      "video/mpeg",
    ],
    "cvUploadMethods": Array [
      "url",
      "file",
    ],
  },
}
`;
